File: mvp/src/main/java/com/mandria/android/mvp/rx/callbacks/OnError.java
package com.mandria.android.mvp.rx.callbacks;

import android.support.annotation.NonNull;

import io.reactivex.functions.BiConsumer;

/**
 * Interface for onError handler.
 */
public interface OnError<U> extends BiConsumer<U, Throwable> {

    @Override
    void accept(@NonNull U u, Throwable throwable) throws Exception;
}

File: core_kotlin/src/main/kotlin/com/mandria/android/mvp/rx/callbacks/OnError.kt
package com.mandria.android.mvp.rx.callbacks

import io.reactivex.functions.BiConsumer


/** Interface for onError handler. **/
interface OnError<U> : BiConsumer<U, Throwable> {

    @Throws(Exception::class)
    override fun accept(u: U, throwable: Throwable)
}